test/new-e2e/tests/orchestrator/suite_test.go

```diff
@@ -0,0 +1,200 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+package orchestrator
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"os"
+	"path"
+	"testing"
+	"time"
+
+	"github.com/pulumi/pulumi/sdk/v3/go/auto"
+	"github.com/stretchr/testify/suite"
+	"gopkg.in/yaml.v3"
+	"k8s.io/client-go/kubernetes"
+	restclient "k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/clientcmd"
+
+	agentmodel "github.com/DataDog/agent-payload/v5/process"
+	"github.com/DataDog/datadog-agent/test/fakeintake/aggregator"
+	fakeintake "github.com/DataDog/datadog-agent/test/fakeintake/client"
+	"github.com/DataDog/datadog-agent/test/new-e2e/pkg/runner"
+	"github.com/DataDog/datadog-agent/test/new-e2e/pkg/utils/infra"
+)
+
+var replaceStacks = flag.Bool("replace-stacks", false, "Attempt to destroy the Pulumi stacks at the beginning of the tests")
+var keepStacks = flag.Bool("keep-stacks", false, "Do not destroy the Pulumi stacks at the end of the tests")
```

Comment:
<!-- Suggestions propose improvements to the current subject. It’s important to be explicit and clear on what is being suggested and why it is an improvement. -->
**💬 suggestion**
You can now use `WithSkipDeleteOnFailure` from the new e2e.Suite

Reply:
Since last wee, you now can you the suite for KindVM tests, we don't have examples but can help with it cc @vboulineau . Not required, can be a follow up.

Reply:
That's great. We have plans to expand the tested feature set, the goal of this work was to get a foundation out there. I will add a note in the following work to migrate to the e2e suite